doc/fwutil/fwutil.md

```diff
@@ -172,18 +187,23 @@ non modular chassis platforms. Both modular and non modular chassis platforms sh
 but may have different implementation.
 
 SONiC FW utility uses platform API to interact with the various platform components.
+SONiC FW utility extends to support for the automatic firmware update based on "platform_components.json" under platform directory and next reboot option which is passed as a option for `fwutil autoupdate fw` command.
+SONiC FW utility also extends to support for the automatic firmware update with a custom firmware package that can include any firmware update tool and the firmware update tool will be used for the firmware update if it's specified in the "platform_components.json".
 
 ## 2.2 FW utility
 
 ### 2.2.1 Command structure
 
 **User interface**:
+
 ```
 fwutil
 |--- show
 |    |--- version
 |    |--- status
 |    |--- updates -i|--image=<current|next>
+|    |--- updates -z|--fw-image=<fw_package.tar.gz>
+|    |--- auto_update_status
```

Comment:
@sujinmkang we need to refactor this part too since moving from `auto-update` to `update all`

Reply:
I think this can be changed to update_status but I hope this doesn't mislead people to think that they could check the  each component update done by `fwutil update {chassis|module <module_name>} component <component_name> fw ...`.
But I agree to change the command.